stock_unfuck/models/res_config.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models
+
+
+class StockConfiguration(models.TransientModel):
+
+    _inherit = 'stock.config.settings'
+
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True,
+        default=lambda self: self.env.user.company_id)
+
+    make_journal_entry = fields.Boolean(
```

Comment:
@GavyMG 

I talked with @hbto and we are agree in keep all this change into product_extended_variants
Could you make the change and move your changes from stock_unfuck to product_extended_variants?